cve_bin_tool/cve_scanner.py

```diff
@@ -50,6 +50,7 @@ class CVEScanner:
     RANGE_UNSET: str = ""
     dbname: str = os.path.join(DISK_LOCATION_DEFAULT, DBNAME)
     CONSOLE: Console = Console(file=sys.stderr, legacy_windows=False)
+    ALPHA_TO_NUM: Dict[str, int] = dict(zip(ascii_lowercase, range(26)))
```

Comment:
If we're only using this in one place, there's no reason to store it as a class thing.  Just put it in the function where it's used.

Reply:
adding: if we're only using it in one place *and* it's unlikely anyone's going to want to configure it.

Reply:
Yes we never need to configure it but I have done that to optimize runtime. If we keep that in function whenever that function get called new dictionary will get allocated in function stack and it will be destroyed after function returns. Since we are going to call this function several times with CVEScanner context it will be more efficient to keep it as a class attribute.